libcloudforensics/gcp.py

```diff
@@ -658,6 +681,83 @@ def ExecuteFunction(self, function_name, args):
     return function_return
 
 
+class GoogleCloudBuild(GoogleCloudProject):
+  """Class to call Google Cloud Build APIs.
+
+  Attributes:
+    gcb_api_client: Client to interact with GCB APIs.
+  """
+  CLOUD_BUILD_API_VERSION = 'v1'
+
+  def __init__(self, project_id):
+    """Initialize the GoogleCloudBuild object.
+
+    Args:
+      project_id (str): The name of the project.
+    """
+
+    self.gcb_api_client = None
+    super(GoogleCloudBuild, self).__init__(project_id)
+
+  def GcbApi(self):
+    """Get a Google Cloud Build service object.
+
+    Returns:
+      apiclient.discovery.Resource: A Google Cloud Build service object.
+    """
+
+    if self.gcb_api_client:
+      return self.gcb_api_client
+    self.gcb_api_client = CreateService(
+        'cloudbuild', self.CLOUD_BUILD_API_VERSION)
+    return self.gcb_api_client
+
+  def CreateBuild(self, build_body):
+    """Create a cloud build.
+    Args:
+      build_body (dict): the build resource describes how to find the source
```

Comment:
build_body (dict): A dictionary that describes how to find the source code and how to build it.

Reply:
done.